src/Framework/TestSuiteIterator.php

```diff
@@ -73,19 +73,26 @@ public function next(): void
 
     /**
      * Returns the sub iterator for the current element.
+     *
+     * @throws \UnexpectedValueException if the current element is no TestSuite
```

Comment:
Exceptions raised by PHPUnit must have the appropriate marker interface. We need something along the lines of
```php
<?php declare(strict_types=1);
namespace PHPUnit\Framework;

final class UnexpectedValueException extends \UnexpectedValueException implements Exception
{
}
```

Reply:
Done and repushed. After this has been merged, I'll create a separate PR for 8.3/master as the new exception class needs to be in a different location there.